service.subtitles.subtitle/service.py

```diff
@@ -125,11 +125,13 @@ def takeTitleFromFocusedItem():
     labelTVShowTitle = xbmc.getInfoLabel("ListItem.TVShowTitle")
     labelSeason = xbmc.getInfoLabel("ListItem.Season")
     labelEpisode = xbmc.getInfoLabel("ListItem.Episode")
+    isItMovie = xbmc.getCondVisibility("Container.Content(movies)") or labelType == 'movie'
```

Comment:
example: link movies to a tvshow, then you get movies in a list with container.content = tvshows. ListItem.Dbtype will be correct in that case, container.content will be misleading.